packages/frontend/src/components/Game/GameBoard/ResultDisplay/index.jsx

```diff
@@ -40,19 +38,21 @@ const ResultDisplay = () => {
             <h2 className="lh-lg">Its a tie!</h2>
           ) : (
             <h2 className={`text-${winner.color.className}`}>
-              {`${winner.name} wins!`}
+              {winner.name }
+               wins!
             </h2>
```

Comment:
This could be cleaned up a bit (I don't think the `lh-lg` class is needed)
```
<h2 className={winner ? `text-${winner.color.className}` : ''}>
  {winner ? `${winner.name} wins!` : "It's a tie!"}
</h2>
```

Reply:
Also, just above this (and I realize it's not really a style thing and wasn't changed in this PR) can we can change `stage ===  2` to `stage === GAME_STAGES.GAME_OVER`? If there are any other places where the stage is referred to as a number instead of using the `GAME_STAGES` object, can those be changed too?

Reply:
@perennialAutodidact for the mobil vs desktop changes, can we merge PR 90 Tiles Extend Off Bottom of Screen? That pr has the get breakpoint hook that we can use to handle mobile size layout configurations.

Reply:
Bootstrap has responsive classes for this, should just be one additional class in the same spot as the one that justifies it to the right

[https://getbootstrap.com/docs/5.0/utilities/flex/#justify-content](https://getbootstrap.com/docs/5.0/utilities/flex/#justify-content)

Reply:
@perennialAutodidact but would't we want to apply that class conditionally depending on what screen size we're at? It would have the right justified class if we're in desktop or tablet size and the center justified class if we're in mobile. 
I thought we were setting up the breakpoint hook for that reason. Let me know if I'm missing something.

Reply:
The breakpoint classes handle it. `justify-content-sm-center` would center on mobile and `justify-content-lg-end` would right justify starting at desktop size. The breakpoint hook was to resize the tiles dynamically based on the screen size or when the screen size changes, such as when the user rotates their phone.